woocommerce/woocommerce-android
# Issue/4437 Add serial number to the metadata of the payment intent

Resolves #4437 

The PR adds a serial number to the metadata of the payment intent

### How to test

Make sure your store is eligible for in-person payments

* Connect to a reader (watch out for this known issue)
* Open detail of an unpaid order in USD
* Tap on the Collect Payment button
* Finish the payment
* Open Stripe's dashboard and search for the payment

![image](https://user-images.githubusercontent.com/4923871/126489280-cedbb891-640b-492b-b701-640f42f66ed4.png)


Update release notes:

[x] I have considered if this change warrants user-facing release notes and have added them to RELEASE-NOTES.txt if necessary.
